package.json

```diff
@@ -1,23 +1,22 @@
 {
   "name": "@hms-dbmi/viv",
   "version": "0.10.4",
-  "main": "dist/index.js",
-  "module": "dist/index.js",
+  "main": "dist/viv.es.js",
+  "module": "dist/viv.es.js",
```

Comment:
I can't figure out how to change the name of the final build from vite. We can always run `mv dist/viv.es.js dist/index.js` if desired.